src/main/flight/position.c

```diff
@@ -94,32 +83,41 @@ static bool altitudeOffsetSetGPS = false;
 void calculateEstimatedAltitude(timeUs_t currentTimeUs)
 {
     static timeUs_t previousTimeUs = 0;
-    static int32_t baroAltOffset = 0;
-    static int32_t gpsAltOffset = 0;
+    static float baroAltOffset = 0;
+    static float gpsAltOffset = 0;
 
-    const uint32_t dTime = currentTimeUs - previousTimeUs;
-    if (dTime < BARO_UPDATE_FREQUENCY_40HZ) {
-        schedulerIgnoreTaskExecTime();
-        return;
-    }
+    uint32_t dTime = currentTimeUs - previousTimeUs;
     previousTimeUs = currentTimeUs;
+    dTime *= 1.0f;
```

Comment:
```suggestion
    uint32_t dTime = (currentTimeUs - previousTimeUs) * 1.0f;
    previousTimeUs = currentTimeUs;
```

should this be a float?